src/components/button/_button.scss

```diff
@@ -44,8 +44,11 @@ $button-border-height: 3px;
   }
 
   // When hovered
-  &:not([class*='btn--ghost']):not([class*='btn--mobile']):not([class*='btn--secondary']):not([class*='btn--print']):hover &__inner {
-    background: darken($color-button, 5%);
+  &:not([class*='btn--ghost']):not([class*='btn--mobile']):not([class*='btn--secondary']):not([class*='btn--print']):not([class*='btn--disabled']):hover
```

Comment:
Is there a better way of doing this rather than listing them all out? Just thinking this is a list we will have to make sure we maintain if we add new button types